backend-tests/backend-tests.hs

```diff
@@ -460,27 +491,34 @@ tests :: FilePath -> TestTree
 tests wasm_file = testGroup "Tests" $ upgradeGroups $
   [ withoutUpgrade $ iiTest "installs" $ \ _cid ->
     return ()
+
   , withoutUpgrade $ iiTest "installs and upgrade" $ \ cid ->
     doUpgrade cid
+
   , withoutUpgrade $ iiTest "register with wrong user fails" $ \cid -> do
-    callIIReject cid dummyUserId #register (device1, invalidPOW)
+    callIIRejectExpectingMessage cid dummyUserId #register (device1, powAt cid 1) "[a-z0-9-]+ could not be authenticated against"
+
   , withoutUpgrade $ iiTest "register with bad pow fails" $ \cid -> do
-    callIIReject cid dummyUserId #register (device1, invalidPOW)
+    callIIRejectExpectingMessage cid webauthID #register (device1, invalidPOW) "proof of work hash check failed"
+
```

Comment:
```suggestion
```

Reply:
Lots of empty newlines introduced, maybe an artefact of your editor?

Reply:
Reverted